Library/Homebrew/utils/inreplace.rb

```diff
@@ -22,7 +22,10 @@ module Inreplace
     def inreplace(paths, before = nil, after = nil, audit_result = true)
       errors = {}
 
-      Array(paths).each do |path|
+      paths = Array(paths)
+      errors["Empty list of files"] = ["no files given to replace"] if paths.empty?
```

Comment:
```suggestion
      errors["`paths` (first) parameter"] = ["`paths` was empty"] if paths.blank?
```

Then you can avoid doing the `Array(paths)` too (and it's a slightly better check)

Reply:
I think the above messaging is also better and sufficient for this to be merged.